deployment/terraform/live/outputs.tf

```diff
@@ -1,7 +1,3 @@
-output "upload_bucket_name" {
-  value = "${module.object_storage.upload_bucket_name}"
-}
```

Comment:
Looking at the config.json now, we don't reference the upload bucket: I had missed that. Thanks for cleaning this up.